### PR TITLE
Ypersky new perf mark

### PR DESCRIPTION
**Pull request by ypersky1980**
Added a new marker : performance_new_feature and marked with it the 3 single  RBD Thick and 1 multiple deletion test cases.  

Signed-off-by: Yulia Persky <ypersky@redhat.com>

### CODE REVIEW

File: pytest.ini [hunk 1]
```diff
@@ -21,6 +21,7 @@ markers =
     workloads: workloads related tests
     flowtests: flowbased related tests
     performance: performance related tests
+    performance_extended: performance related tests for RBD_Thick
```
Comment (keesturam) [inline pytest.ini:24]: `performance related tests for RBD_Thick`
This definition is misleading. we have performance_extended for any performance tests that need not be part of regular regression, but can be run at will if required.

**Review by keesturam — CHANGES_REQUESTED** [review]

File: tests/e2e/performance/test_pvc_creation_deletion_performance.py [hunk 2]
```diff
@@ -384,7 +384,7 @@ def write_file_on_pvc(self, pvc_obj, filesize=1):
             ),
             pytest.param(
                 *[constants.CEPHBLOCKPOOL_THICK],
-                marks=[pytest.mark.performance],
+                marks=[pytest.mark.performance, pytest.mark.performance_new_feature],
```
Comment (ramkiperiy) [inline tests/e2e/performance/test_pvc_creation_deletion_performance.py:387]: @ypersky1980 forgot to change line 387?

File: ocs_ci/framework/pytest_customization/marks.py [hunk 3]
```diff
@@ -58,6 +58,7 @@
 workloads = pytest.mark.workloads
 flowtests = pytest.mark.flowtests
 performance = pytest.mark.performance
+performance_extended = pytest.mark.performance_extended
```
Comment (petr-balogh) [inline ocs_ci/framework/pytest_customization/marks.py:61]: This new marker should also be defined here:
https://github.com/red-hat-storage/ocs-ci/blob/master/pytest.ini#L2

File: tests/e2e/performance/test_pvc_creation_deletion_performance.py (hunk 2 above)
Comment (ypersky1980) [reply tests/e2e/performance/test_pvc_creation_deletion_performance.py:387]: Done

File: pytest.ini (hunk 1 above)
Comment (ypersky1980) [reply pytest.ini:24]: @keesturam changed to : 

performance_extended: non regression performance tests

File: ocs_ci/framework/pytest_customization/marks.py (hunk 3 above)
Comment (ypersky1980) [reply ocs_ci/framework/pytest_customization/marks.py:61]: @petr-balogh   
Added ( line 24) 

23    performance: performance related tests
24    performance_extended: non regression performance tests
25   scale: scale related tests

**Review by keesturam — APPROVED** [review]
LGTM

**Review by Avilir — APPROVED** [review]
LGTM

**Comment by ypersky1980** [comment]
The test log for validation run can be found here: /home/ypersky/ypersky_pr_validation/pr_4552/lr5-ypersky-perf444_test_pvc_creation_deletion_perf_1505_1

11 out of 12 tests passed, and the reason for the 1 failure is not related to the markers : 

10:06:22 - MainThread - ocs_ci.ocs.resources.pod - INFO - Waiting for FIO results from pod pod-test-rbd-76a7b1e90ef740d08f80ab80edf
10:06:32 - MainThread - ocs_ci.ocs.resources.pod - ERROR - Found Exception: while parsing a flow node

**Pull request merged**